cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -194,3 +202,19 @@ func (p PasstNetworkConfigurator) generatePortForward() []domainschema.Interface
 
 	return portsFwd
 }
+
+func (p PasstNetworkConfigurator) discoverSourceLinkName() string {
+	// optionalLinkName link name to look for before falling back to eth0 if the
+	// link do not exist
+	// FIXME: This will be configurable in the future.
+	const (
+		optionalLinkName = "ovn-udn1"
+	)
+
+	sourceLinkName := optionalLinkName
+	_, err := p.handler.LinkByName(sourceLinkName)
+	if err != nil {
```

Comment:
I think you need to check for the specific error, otherwise we may mask real problems.
Is it possible?